bin/kyuubibulid.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+set -x
+set -e
+#cm_ext branch
+CM_EXT_BRANCH=cm5-5.15.0
+kyuubi_version=1.6.1
+parcel_version=${kyuubi_version}-1
+kyuubi_package_url=https://archive.apache.org/dist/incubator/kyuubi/kyuubi-${kyuubi_version}-incubating/apache-kyuubi-${kyuubi_version}-incubating-bin.tgz
```

Comment:
`-incubating` should be part of version, and since Kyuubi 1.7.0 has been released, we can move to 1.7.0 instead.

Reply:
Ok, thank you for your reply. Please check again